### PR TITLE
vSphere: add missing fields in example

### PR DESCRIPTION
**Pull request by kron4eg**
Signed-off-by: Artiom Diomin <kron82@gmail.com>

**What this PR does / why we need it**:
Example didn't reflected all the possible config option accessible in vSphere

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

```release-note
vSphere: add missing fields in example
```

### CODE REVIEW

File: examples/vsphere-machinedeployment.yaml [hunk 1]
```diff
@@ -37,20 +37,25 @@ spec:
...
             password:
               secretKeyRef:
                 namespace: kube-system
                 name: machine-controller-vsphere
                 key: password
-            cluster: "test-cluster"
+            datacenter: datacenter1
+            templateVMName: ubuntu-template
+            # Optional
+            templateNetName: ''
```
Comment (alvaroaleman) [inline examples/vsphere-machinedeployment.yaml]: Could you add an explanation, that `templateNetName` is the name of the network the interface on the template-machine is attached to that should be used for the `vmNetName` and also that its only required if the template-machine has more than one network?

File: examples/vsphere-machinedeployment.yaml [hunk 2]
```diff
@@ -37,20 +37,25 @@ spec:
...
+            vmNetName: network1
```
Comment (alvaroaleman) [inline examples/vsphere-machinedeployment.yaml]: This is also optional

File: examples/vsphere-machinedeployment.yaml [hunk 3]
```diff
@@ -37,20 +37,25 @@ spec:
...
+            # Optional
+            folder: folder1
```
Comment (alvaroaleman) [inline examples/vsphere-machinedeployment.yaml]: Are you sure, does it work when this is unset?

Comment (kron4eg) [reply examples/vsphere-machinedeployment.yaml]: Quite sure, in kubeone example I didn't used it and it's all good: https://github.com/kubermatic/kubeone/blob/master/examples/terraform/vsphere/outputs.tf#L57

File: examples/vsphere-machinedeployment.yaml (hunk 1 above)
Comment (kron4eg) [reply examples/vsphere-machinedeployment.yaml]: done

File: examples/vsphere-machinedeployment.yaml (hunk 3 above)
Comment (kron4eg) [reply examples/vsphere-machinedeployment.yaml]: done

File: examples/vsphere-machinedeployment.yaml (hunk 2 above)
Comment (kron4eg) [reply examples/vsphere-machinedeployment.yaml]: done

**Comment by kron4eg** [comment]
@alvaroaleman we can /override it

**Pull request merged**